questionnaire.js

```diff
@@ -772,87 +773,77 @@ export function nextClick(norp, retrieve, store, rootElement) {
 }
 
 function setNumberOfQuestionsInModal(num, norp, retrieve, store, soft) {
-  let prompt = `There ${num > 1 ? "are" : "is"} ${num} question${num > 1 ? "s" : ""
-    } unanswered on this page. `;
-  if (!soft) {
-
-    // set modal text...
-    document.getElementById(
-      "hardModalBodyText"
-    ).innerText = `${prompt} Please answer the question${num > 1 ? "s" : ""}.`;
-
-      // popup the modal..
-    const hardModal = new bootstrap.Modal(document.getElementById('hardModal'))
-    hardModal.show()
-    return null;
+  const prompt = `There ${num > 1 ? "are" : "is"} ${num} unanswered question${num > 1 ? "s" : ""} on this page.`;
```

Comment:
Sure, these prompts are back to their original language. Nothing changed in the look or loading sequence, just accessible navigation, so that should be all set.